src/components/Common/BadgeMobile/index.tsx

```diff
@@ -0,0 +1,83 @@
+import { forwardRef } from 'react';
+
+interface BadgeMobileProps {
+  badgeType?:
+    | 'default'
+    | 'primary'
+    | 'primary4K'
+    | 'danger'
+    | 'danger4K'
+    | 'warning'
+    | 'warning4K'
+    | 'success'
+    | 'success4K';
+  className?: string;
+  href?: string;
+  children: React.ReactNode;
```

Comment:
We won't need any of these anymore. Let's also drop the forwardRef that is wrapping this component.